tests/test_tools.py

```diff
@@ -195,6 +197,121 @@ def test_set_output_directory_bad(user_filesystem):
         assert not Path(actual_args.output_directory).is_dir()
 
 
+@pytest.mark.parametrize(
+    "inputs, expected",
+    [
+        # Test when only a home config file exists (no local config file),
+        # expect to return args if wavelength or anode type is specified,
+        # otherwise update args with values from the home config file.
+        # C1: no args, expect to update arg values from home config
+        ([""], {"wavelength": 0.3, "anode_type": None}),
+        # C2: wavelength provided, expect to return args unchanged
+        (["--wavelength", "0.25"], {"wavelength": 0.25, "anode_type": None}),
+        # C3: anode type provided, expect to return args unchanged
+        (["--anode-type", "Mo"], {"wavelength": None, "anode_type": "Mo"}),
+        # C4: both wavelength and anode type provided,
+        # expect to return args unchanged
```

Comment:
Is this what we want?  I think it should be either `wavelength` or `anode_type`.  If both are provided raise an exception.